### ISSUE TITLE
Vorlon Keyboard Accessibility

### ISSUE THREAD

**Issue by marcysutton** [MicrosoftDX/Vorlonjs#332]
When I try to use the keyboard with Vorlon.js, I can't due to the many DIV elements used for interface buttons. Anything that a user clicks on should be reachable from the keyboard and a screen reader. This can be accomplished easiest by using `<button>` elements with some kind of text in them. If CSS complicates changing the element, you can accomplish similar behavior using `<div tabindex="0" role="button">` with keyboard events bound (versus buttons, which get implicit tabindex, role and keyboard event on click for free).

For button text (which screen readers need), if the buttons have visible textContent in them, that will be sufficient. However, if they are icon buttons with no text, you can use the `aria-label` attribute directly on the button.

![vorlon main tabs](https://cloud.githubusercontent.com/assets/1045233/13673640/e000516c-e68e-11e5-9ef6-4f2ce3933043.png)

The problem also exists on the Best Practices tabs:
![vorlon best practices tabs](https://cloud.githubusercontent.com/assets/1045233/13673641/e00a2368-e68e-11e5-96a8-091d7f53e504.png)

An aXe audit also found an issue with the `<x-action>` buttons, which have positive tabindex numbers but no button role or text in them.
